src/main/java/mate/academy/Main.java

```diff
@@ -6,15 +6,21 @@
 import mate.academy.service.AuthenticationServiceImpl;
 import mate.academy.service.OrderService;
 import mate.academy.service.OrderServiceImpl;
+import org.apache.logging.log4j.LogManager;
+import org.apache.logging.log4j.Logger;
 
 public class Main {
+    private static final Logger logger = LogManager.getLogger(Main.class);
+    private static final String LOGIN = "bob";
+
     public static void main(String[] args) {
         AuthenticationService authenticationService = new AuthenticationServiceImpl();
         User user;
         try {
-            user = authenticationService.login("bob", "1234");
+            user = authenticationService.login(LOGIN, "1234");
         } catch (AuthenticationException e) {
-            e.printStackTrace();
+            logger.error("Authentication exception. Params: login= {}", LOGIN, e);
+            System.out.println("hello");
```

Comment:
The 'System.out.println' is not informative for logging purposes. Consider using logger with an informative message to log the occurrence of an AuthenticationException.